gatsby-config.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config({
+  path: `.env.${process.env.NODE_ENV}`,
+})
+
+const strapiConfig = {
+  apiURL: process.env.STRAPI_API_URL,
+  accessToken: process.env.STRAPI_TOKEN,
+  collectionTypes: ['product'],
+  singleTypes: [],
+}
+
+module.exports = {
+  siteMetadata: {
+    title: `boilerplate-gatsby-strapi`,
+    siteUrl: `https://www.yourdomain.tld`,
```

Comment:
Can we set a bit more realistic values?

We should try to host it on marketing-website-demo.jalantechnologies.com if possible.

If subdomain is not possible, try jalantechnologies.com/marketing-website-demo

If that's not possible, maybe try new domain